src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/dto/DraftReferralDTO.kt

```diff
@@ -19,6 +20,7 @@ data class DraftReferralDTO(
   val hasAdditionalResponsibilities: Boolean? = null,
   val whenUnavailable: String? = null,
   val additionalRiskInformation: String? = null,
+  val desiredOutcomes: List<DesiredOutcome>? = null
```

Comment:
the type of this field should be a `List<UUID>` and the field name `desiredOutcomesIds` as per the contract test spec.

Reply:
Updated